MatrixSDK/Space/MXSpaceService.swift

```diff
@@ -87,6 +108,114 @@ public class MXSpaceService: NSObject {
         let room = self.session.room(withRoomId: spaceId)
         return room?.toSpace()
     }
+        
+    /// Get the space children informations of a given space from the server.
+    /// - Parameters:
+    ///   - spaceId: The room id of the queried space.
+    ///   - parameters: Space children request parameters.
+    ///   - completion: A closure called when the operation completes.
+    /// - Returns: a `MXHTTPOperation` instance.
+    @discardableResult
+    public func getSpaceChildrenForSpace(withId spaceId: String,
+                                         parameters: MXSpaceChildrenRequestParameters?,
+                                         completion: @escaping (MXResponse<MXSpaceChildrenSummary>) -> Void) -> MXHTTPOperation {
+        return self.session.matrixRestClient.getSpaceChildrenForSpace(withId: spaceId, parameters: parameters) { (response) in
+            switch response {
+            case .success(let spaceChildrenResponse):
+                guard let rooms = spaceChildrenResponse.rooms else {
+                    // We should have at least one room for the requested space
+                    completion(.failure(MXSpaceServiceError.spaceNotFound))
+                    return
+                }
+
+                guard let rootSpaceChildSummaryResponse = rooms.first(where: { spaceResponse -> Bool in
+                    return spaceResponse.roomId == spaceId
+                }) else {
+                    // Fail to find root child. We should have at least one room for the requested space
+                    completion(.failure(MXSpaceServiceError.spaceNotFound))
+                    return
+                }
+
+                // Build the queried space summary
+                let spaceSummary = self.createRoomSummary(with: rootSpaceChildSummaryResponse)
+
+                // Build the child summaries of the queried space
+                let childInfos = self.spaceChildInfos(from: spaceChildrenResponse, excludedSpaceId: spaceId)
+
+                let spaceChildrenSummary = MXSpaceChildrenSummary(spaceSummary: spaceSummary, childInfos: childInfos)
+
+                completion(.success(spaceChildrenSummary))
+            case .failure(let error):
+                completion(.failure(error))
+            }
+        }
+    }
+    
+    // MARK: - Private
+    
+    private func createRoomSummary(with spaceChildSummaryResponse: MXSpaceChildSummaryResponse) -> MXRoomSummary {
+        
+        let roomId = spaceChildSummaryResponse.roomId
+        let roomTypeString = spaceChildSummaryResponse.roomType
+                
+        let roomSummary: MXRoomSummary = MXRoomSummary(roomId: roomId, andMatrixSession: nil)
+        roomSummary.roomTypeString = roomTypeString
+        roomSummary.roomType = self.roomTypeMapper.roomType(from: roomTypeString)
+        
+        let membersCount = MXRoomMembersCount()
```

Comment:
What about `MXRoomMembersCount.members` and `MXRoomMembersCount.invited`?
We should have `members >= joined`.